providers/bulksms/package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "@novu/bulksms",
+  "version": "0.12.0",
+  "description": "A bulksms wrapper for novu",
+  "main": "build/main/index.js",
+  "typings": "build/main/index.d.ts",
+  "module": "build/module/index.js",
+  "private": false,
+  "repository": "https://github.com/novuhq/novu",
+  "license": "MIT",
+  "keywords": [],
+  "scripts": {
+    "prebuild": "rimraf build",
+    "build": "run-p build:*",
+    "build:main": "tsc -p tsconfig.json",
+    "build:module": "tsc -p tsconfig.module.json",
+    "fix": "run-s fix:*",
+    "fix:prettier": "prettier \"src/**/*.ts\" --write",
+    "fix:lint": "eslint src --ext .ts --fix",
+    "test": "run-s build test:*",
+    "test:lint": "eslint src --ext .ts",
+    "test:unit": "jest src",
+    "watch:build": "tsc -p tsconfig.json -w",
+    "watch:test": "jest src --watch",
+    "reset-hard": "git clean -dfx && git reset --hard && yarn",
+    "prepare-release": "run-s reset-hard test"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "engines": {
+    "node": ">=13.0.0 <17.0.0",
+    "pnpm": "^7.26.0"
+  },
+  "dependencies": {
+    "@novu/stateless": "^0.12.0"
```

Comment:
Looks like you are usign node-fetch as a fallback, but i can't see it in the dependency list here. Could you please add it?

Reply:
@scopsy added pls check